packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -491,7 +491,7 @@ const RealAccountSignup = ({
                 active_modal_index_no = modal_pages_indices.add_or_manage_account;
             } else {
                 active_modal_index_no =
-                    (!currency && real_account_signup_target !== 'maltainvest') || is_loading
+                    real_account_signup_target === 'set_currency'
```

Comment:
real_account_signup_target is usually landing company. isnt? Do we have "'set_currency" has real_account_signup_target?

Reply:
yes I just checked and verified with account creation and manage account as well 

`['set_currency', 'manage'].includes(this.root_store.ui.real_account_signup_target)`

Removed currency as it was setting the currency in the store as well which was making the condition false